canvas/src/main/scala/doodle/canvas/algebra/Raster.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2015 Creative Scala
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package doodle.canvas.algebra
+
+import doodle.algebra.Algebra
+import doodle.algebra.generic._
+
+trait Raster extends GenericRaster[CanvasDrawing, Unit] {
```

Comment:
I'm pretty sure `Unit` is the wrong type to use here. Don't you want `CanvasRenderingContext2D`?